programs/token/src/instructions/initialize_multisig.rs

```diff
@@ -0,0 +1,120 @@
+use core::{mem::MaybeUninit, slice};
+
+use pinocchio::{
+    account_info::AccountInfo,
+    cpi::slice_invoke_signed,
+    instruction::{AccountMeta, Instruction, Signer},
+    program_error::ProgramError,
+    ProgramResult,
+};
+
+/// Initialize a new Multisig.
+///
+/// ### Accounts:
+///   0. `[writable]` The multisig account to initialize.
+///   1. `[]` Rent sysvar
+///   2. ..`2+N`. `[]` The N signer accounts, where N is between 1 and 11.
+pub struct InitializeMultisig<'a> {
```

Comment:
Another lifetime for the array of signers:
```suggestion
pub struct InitializeMultisig<'a, 'b> {
```